queries.md

```diff
@@ -2,20 +2,43 @@
 
 ## find all customers that live in London. Returns 6 records.
 
+SELECT \* FROM [Customers]
```

Comment:
I'm wondering why you're *s are preceded by \. That syntax is invalid to my knowledge. The queries look correct otherwise!